Header header
geometry_msgs/Pose pose
geometry_msgs/Twist twist
geometry_msgs/Vector3 gyro_bias
geometry_msgs/Vector3 accel_bias

float64[324] covariance